src/gamestate/system_state.cpp

```diff
@@ -2295,6 +2295,9 @@ void state::load_scenario_data(parsers::error_handler& err) {
 		if(!bool(military_definitions.irregular)) {
 			err.accumulated_errors += "No irregular (or equivalent unit type) found\n";
 		}
+		if(!bool(military_definitions.artillery)) {
+			err.accumulated_errors += "No artillery (or equivalent unit type) found\n";
```

Comment:
this will always trigger, I expect, because artillery isn't identified as part of loading the unit files. It is identified as part of the load unsaved data routines